Datagrid/Datagrid.php

```diff
@@ -139,9 +134,9 @@ function ($value) {
             }
         ));
 
-        $this->formBuilder->add('_sort_order', $hiddenType);
-        $this->formBuilder->add('_page', $hiddenType);
-        $this->formBuilder->add('_per_page', $hiddenType);
+        $this->formBuilder->add('_sort_order', 'Symfony\Component\Form\Extension\Core\Type\HiddenType');
+        $this->formBuilder->add('_page', 'Symfony\Component\Form\Extension\Core\Type\HiddenType');
+        $this->formBuilder->add('_per_page', 'Symfony\Component\Form\Extension\Core\Type\HiddenType');
```

Comment:
as we depend on PHP 7.1, why don't we use `::class` constant instead of FQCN strings?

Reply:
Because StyleCI will remove this...

Reply:
In theory yes, Im waiting for StyleCI to do it, but I am not sure about it, did we tried that for any sonata bundle?

Reply:
> Because StyleCI will remove this...

I know this @core23 , but why don't we use `::class` constants and allow/define the rule in our StyleCI/PHP-CS-Fixer config?

Reply:
I will change it

Reply:
> I know this @core23 , but why don't we use `::class` constants and allow/define the rule in our StyleCI/PHP-CS-Fixer config?

Afaik we do this, because we had PHP 5.3 support, but we should allow this for the next majors. Not sure if we need two .styleci configs for this (stable, next major).